torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -124,6 +124,54 @@ Node* CreateQuantizedBias(
   return const_node;
 }
 
+std::vector<Node*> CreateQuantizedWeightsONNX(
```

Comment:
If not specified, it makes sense that most functions in this file are for standard ONNX, so naming suggestion:

```suggestion
std::vector<Node*> CreateQuantizedWeights(
```